build.gradle

```diff
@@ -798,8 +853,8 @@ subprojects {
     skipProjects = [ ":jmh-benchmarks", ":trogdor" ]
     skipConfigurations = [ "zinc" ]
   }
-
-  if (project.path in spotlessApplyModules) {
+  
+  if(JavaVersion.current().isJava11Compatible() && project.path !in excludedSpotlessModules) {
```

Comment:
Maybe we can just throw exceptions if the current JRE is not Java 11 compatible.